web/wp-content/themes/bumblebee/archive.php

```diff
@@ -49,31 +49,77 @@ function () {
 		);
 		?>
 	</section>
-	<?php if ( have_posts() ) : ?>
-		<section class="archive-content">
-			<div class="archive-headings">
-				<div class="breadcrumbs">
-				<?php if ( function_exists( 'yoast_breadcrumb' ) ) : ?>
-					<?php yoast_breadcrumb( '<p id="breadcrumbs">', '</p>' ); ?>
-				<?php endif; ?>
-				</div>
-				<?php the_archive_title( '<h1 class="page-title">', '</h1>' ); ?>
-				<?php the_archive_description( '<div class="archive-description">', '</div>' ); ?>
+	<section class="archive-content">
+		<div class="archive-headings">
+			<div class="breadcrumbs">
+			<?php if ( function_exists( 'yoast_breadcrumb' ) ) : ?>
+				<?php yoast_breadcrumb( '<p id="breadcrumbs">', '</p>' ); ?>
+			<?php endif; ?>
 			</div>
+			<?php the_archive_title( '<h1 class="page-title">', '</h1>' ); ?>
+			<?php the_archive_description( '<div class="archive-description">', '</div>' ); ?>
+		</div>
+	</section>
+	<section class="archive-content">
 
-			<?php the_post(); ?>
-			<?php get_template_part( 'template-parts/archive/content', 'hero' ); ?>
-			<ul class="featured-posts">
-				<?php
+		<?php
+			// Hero post
+		if ( have_posts() ) {
+			the_post();
+			get_template_part( 'template-parts/archive/content', 'hero' );
+		}
+		?>
+		<ul class="featured-posts">
+			<?php
+				// Row of 5 posts
 				$i = 0;
-				while ( have_posts() && $i++ < 5 ) :
-					the_post();
-					get_template_part( 'template-parts/archive/content', 'featured' );
-					endwhile;
-				?>
-			</ul>
+			while ( have_posts() && $i++ < 5 ) :
+				the_post();
+				get_template_part( 'template-parts/archive/content', 'featured' );
+				endwhile;
+			?>
+		</ul>
+		<section class="ad">
+			<?php
+			$slot_name = 'scroll';
+			$tf_slot   = 'btf';
+			if ( 1 === $section_num ) {
+				$slot_name = 'top';
+				$tf_slot   = 'atf';
+			} elseif ( 2 === $section_num ) {
+				$slot_name = 'middle';
+				$tf_slot   = 'atf';
+			}
+			bumblebee_render_ad(
+				uniqid( 'ad' ),
+				[
+					'slot-name'        => $slot_name,
+					'responsive-sizes' => [
+						'mobile'       => [ [ 300, 250 ], [ 320, 50 ], [ 3, 3 ] ],
+						'tablet'       => [ [ 300, 250 ], [ 320, 50 ], [ 3, 3 ] ],
+						'desktop'      => [ [ 728, 90 ], [ 300, 250 ], [ 3, 3 ] ],
+						'large_screen' => [ [ 970, 550 ], [ 970, 250 ], [ 970, 90 ], [ 728, 90 ], [ 300, 250 ], [ 3, 3 ] ],
+					],
+					'targeting'        => [
+						'tf'       => $tf_slot,
+						'pos'      => $slot_name,
+						'location' => $slot_name,
+					],
+				]
+			);
+			?>
 		</section>
-		<section class="full-width-ad">
+		<div class="pure-g recipes">
```

Comment:
I know the `recipes` class was here before... but why? CPT doesn't have recipes.

Reply:
Created ticket https://readersdigest.atlassian.net/browse/DRDT-118 to address such issues